bedrock/firefox/urls.py

```diff
@@ -34,7 +34,7 @@
         views.all_downloads, name='firefox.all'),
     page('firefox/channel', 'firefox/channel.html'),
     redirect('^firefox/channel/android/$', 'firefox.channel', locale_prefix=False),
-    url('firefox/choose', views.choose, name='firefox.choose'),
+    url(r'^firefox/choose/$', views.choose, name='firefox.choose'),
```

Comment:
trailing slashes aren't optional in urls.py files. it is optional in global.conf.